packages/app/src/app/components/Preview/DevTools/Tests/index.tsx

```diff
@@ -25,24 +23,26 @@ export type Status = 'idle' | 'running' | 'pass' | 'fail';
 
 export type TestError = Error & {
   matcherResult?: boolean;
-  mappedErrors?: Array<{
+  mappedErrors?: {
     fileName: string;
     _originalFunctionName: string;
     _originalColumnNumber: number;
     _originalLineNumber: number;
-    _originalScriptCode: Array<{
-      lineNumber: number;
-      content: string;
-      highlight: boolean;
-    }> | null;
-  }>;
+    _originalScriptCode:
+      | {
+          lineNumber: number;
+          content: string;
+          highlight: boolean;
+        }[]
+      | null;
+  }[];
 };
 
 export type Test = {
-  testName: Array<string>;
-  duration: number | undefined;
+  testName: string[];
+  duration?: number;
```

Comment:
Let's keep this `duration: number | undefined`. Like I said before, it changes whether the key is set on the object or not (in this case it's always set, but can be `undefined`);

```js
const a = { test: undefined }; // `('test' in a) === true
const a = { }; // `('test' in a) === false
```

Reply:
I understand the difference when you test those 2, but if I put  `duration?: number`, WebStorm gives me `duration:  number | undefined`, so these 2 are the same I think.